schemas/operators/accessor/variable.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
I hope this schema implements all features documented here:
http://jsonlogic.com/operations.html#var